tests/sqlalchemy_dialect_compliance/test_dialect_compliance.py

```diff
@@ -279,13 +772,19 @@ def test_select_recursive_round_trip(self):
         pass
 
 
-class ComponentReflectionTest(_ComponentReflectionTest):
-    @pytest.mark.skip("Big query types don't track precision, length, etc.")
-    def course_grained_types():
-        pass
+del ComponentReflectionTest  # Multiple tests re: CHECK CONSTRAINTS, etc which
+# BQ does not support
+# class ComponentReflectionTest(_ComponentReflectionTest):
```

Comment:
IF we are ditching this test, then let's delete OR remove the comment octothorps if the test is still valid/needed.

Reply:
Will be removing this and cleaning up this file in #987